.zenodo.json

```diff
@@ -0,0 +1,23 @@
+{
+  "creators": [
+    {
+      "affiliation": "Kitware, Inc",
+      "name": "McCormick, Matthew",
+      "orcid": "0000-0001-9475-3756"
+    }
+  ],
```

Comment:
```
    {
      "affiliation": "University of Iowa",
      "name": "Johnson, Hans",
      "orcid": "0000-0001-9513-2660"
    }
```

Reply:
{
      "affiliation": "Universit\u00e9 de Sherbrooke",
      "name": "Legarreta, Jon Haitz",
      "orcid": "0000-0002-9661-1396"
    }

Not sure whether we are willing to separate the original *creators* of the toolkit vs. *contributors*. It is apparently possible according to the zenodo documentation: https://developers.zenodo.org/, but the *contributors* tag description does not fully match what a contributor is in terms of software/the toolkit.

Reply:
>  "orcid": "0000-0001-9513-2660"

>  "orcid": "0000-0002-9661-1396"

@hjmjohnson got these added.

>  contributors tag description does not fully match what a contributor is in terms of software/the toolkit.

Yes, *contributors* seems more appropriate for a small, confined project where people are full time with dedicated roles as described versus a community-driven project like ITK.